app/contributing/index.md

```diff
@@ -0,0 +1,680 @@
+
+features:
+  - title: "`AccessRole`"
+    description: Specifies the access and resources that are granted.
+    global_scope: true
+  - title: "`AccessRoleBinding`"
+    description: |
+      Assigns a set of `AccessRoles` to a set of objects (users and groups). 
+    global_scope: true
+
+{% endfeature_table %}{% endraw %}
+```
+<!--vale on-->
+
+Rendered output:
+<!--vale off-->
+{% feature_table %} 
+item_title: Mesh RBAC Role
+columns:
+  - title: Description
+    key: description
+  - title: Scoped Globally
+    key: global_scope
+
+features:
+  - title: "`AccessRole`"
+    description: Specifies the access and resources that are granted.
+    global_scope: true
+  - title: "`AccessRoleBinding`"
+    description: |
+      Assigns a set of `AccessRoles` to a set of objects (users and groups). 
+    global_scope: true
+
+{% endfeature_table %}
+<!--vale on-->
+
+### kong.conf table
+
+Renders a list of `kong.conf` parameters into a table.
+
+Code example:
+<!--vale off-->
+```
+{% raw %}{% kong_config_table %}
+config:
+  - name: proxy_server
+  - name: proxy_server_ssl_verify
+  - name: cluster_use_proxy
+  - name: lua_ssl_trusted_certificate
+{% endkong_config_table %}{% endraw %}
+```
+<!--vale on-->
+
+Rendered output:
+<!--vale off-->
+{% kong_config_table %}
+config:
+  - name: proxy_server
+  - name: proxy_server_ssl_verify
+  - name: cluster_use_proxy
+  - name: lua_ssl_trusted_certificate
+{% endkong_config_table %}
+<!--vale on-->
+
+### Tab groups
+
+You can use `navtabs` to create tabs with different labels.
+
+Code example:
+<!--vale off-->
+```
+{% raw %}{% navtabs "deploy-a-license" %}
+{% navtab "Admin API" %}
+
+text
+
+{% endnavtab %}
+{% navtab "license.json" %}
+
+text
+
+{% endnavtab %}
+{% endnavtabs %}{% endraw %}
+```
+<!--vale on-->
+
+Rendered output:
+<!--vale off-->
+{% navtabs "deploy-a-license" %}
+{% navtab "Admin API" %}
+
+text
+
+{% endnavtab %}
+{% navtab "license.json" %}
+
+text
+
+{% endnavtab %}
+{% endnavtabs %}
+<!--vale on-->
+
+### {{site.konnect_short_name}} roles
+
+Creates a table with the {{site.konnect_short_name}} roles and descriptions.
+<!--vale off-->
+```
+{% raw %}{% konnect_roles_table %}
+schema: control_planes
+{% endkonnect_roles_table %}{% endraw %}
+```
+
+### Version badge
+
+Displays which version a feature was introduced in. You can use this in headers, tables, and inline text.
+
+Code example:
+<!--vale off-->
+```
+{% raw %}{% new_in 3.8 %} New feature{% endraw %}
+```
+<!--vale on-->
+
+Rendered output:
+<!--vale off-->
+{% new_in 3.8 %} New feature
+<!--vale on-->
+
+### No copy code
+
+Prevents users from copying code in a code block.
+<!--vale off-->
+```
+{% raw %}{:.no-copy-code}{% endraw %}
+```
+<!--vale on-->
+
+### Render code block/text per deployment type
+
+Displays the code block/text only if the specified deployment type is selected.
+<!--vale off-->
+```
+{% raw %}{: data-deployment-topology="konnect" }
+{: data-deployment-topology="on-prem" }{% endraw %}
```

Comment:
````suggestion
{% raw %}
```
code for konnect
```
{: data-deployment-topology="konnect" }

```
code for on-prem
```
{: data-deployment-topology="on-prem" }{% endraw %}
````
I _think_ using the `raw` tag works on backticks, but I'm not 100% sure.